src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -160,7 +168,7 @@ public void testFromXContent() throws IOException {
     public void testToQuery() throws IOException {
         testQuery = createTestQueryBuilder();
         QueryParseContext context = createContext();
-        context.setMapUnmappedFieldAsString(true);
+        context.setAllowUnmappedFields(true);
```

Comment:
With the current setup we test unmapped fields when the random setup sets to/from to numeric fields, and mapped fields if we have date string for to/from. Does this make sense of do we need to extend this?